src/DummyConsoleApp/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+//[assembly: AssemblyVersion("1.0.0.0")]
+//[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
In my .net code conversion attempts, I typically try to completely delete the properties directory.  It's rarely useful.

And if you do need it, you almost certainly need virtually no attributes: certainly everything you left behind here is unnecessary.